### ISSUE TITLE
Update Debian package spec to avoid Oracle Java 8 (it's no longer supported)

### ISSUE THREAD

**Issue by rnazaryan** [tobyweston/temperature-machine#89]
Using sudo apt-get install temperature-machine tries to install oracle-java8-jdk, but Oracle stopped the public release of java 8 back in May...could other versions of java be supported?

**Comment by tobyweston** [comment]
Should be an easy fix, openJDK 8 or potentially higher. I'll comment here when I figure out (remind myself) what needs to be done but I imagine it's a change to the release bundle / package spec (`package.sbt`).

**Comment by tobyweston** [comment]
The ❌above on a334068 means, it no like oracle 9

**Comment by tobyweston** [comment]
Some context: I want to see if we can build locally and on CI with the following.

 - [x] Oracle JDK 8
 - [x] OpenJDK 8
 - [ ] OpenJDK 11

to give the option to just switch over. But then I'd like to see if there would be any problems going up through the versions, up to 11. 

Although I think the most sensible leap (at least with the Scala aspect is to go to Java 11, skipping 9, and 10 (see https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html), it's not clear to me yet if there'd be any unforeseen surprises with Raspian.

If I'm reading the recent change with Oracle licencing correctly and the LTS view being only on 11, again we should probably skip 9 and 10.

**Comment by tobyweston** [comment]
Nothing is easy, looks like the version of RRD4J we use doesn't support Java >8.

https://github.com/openhab/openhab1-addons/issues/5663

3.3.1 (https://github.com/openhab/openhab2-addons/pull/4106) would fix it but previous API breaking changes make that difficult for us. This would be a yak shave and I'd much rather move away from RRD to Influx #73 anyway.

**Tl;DR** to support Java 11, first upgrade RRD4J 🤢

**Comment by tobyweston** [comment]
As expected CI reports we can't run against Java 11 ❌ 

See https://travis-ci.org/tobyweston/temperature-machine/jobs/551020957

**Comment by tobyweston** [comment]
This is fixed now. 👍 

When you do an update be sure to use the following command.

```
sudo apt-get update  --allow-insecure-repositories
```

This is required until I fix #84

**Comment by DemBones78** [comment]
hey i know it's only been a year and a half, but can you update the install doc to reflect this "temporary" fix

**Comment by Jurgen1234** [comment]
Hi - A very noobie. I have all downloaded and updated, but when I run temperature-machine --init, it says I need to update java. Done that, but no go. I need step by step instructions to get the java working and going. Hardware all done, w1_slave file there. Above info is beyond me. Running "Raspbian GNU/Linux 10 (buster)".

**Comment by tobyweston** [comment]
Can you give some more details? What version of Java was installed and what did it want to upgrade to? I'd tested on Buster but am curious if I missed something. The install should install the correct version of Java (OpenJDK 8).
